### ISSUE TITLE
Missing libboost-python3

### ISSUE THREAD

**Issue by forresti** [belltailjp/selective_search_py#6]
When I do `cmake` and then `make`, I get the following output:

```
[100%] Building CXX object CMakeFiles/segment.dir/segment_py.cpp.o
cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++ [enabled by default]

Linking CXX shared library segment.so
/usr/bin/ld: cannot find -lboost_python3
collect2: error: ld returned 1 exit status
```

**What is libboost_python3, and where do I obtain it?**

---

Other notes:
- I installed Boost.NumPy, and I'm not having errors with that.
- I'm using Ubuntu 14.04
- I have already installed the obvious boost packages: libboost-python-dev, libboost-all-dev

**Comment by forresti** [comment]
I think I solved it.

we already have this installed:

```
/usr/lib/x86_64-linux-gnu/libboost_python-py34.so
```

so, I did...

```
cd /usr/lib/x86_64-linux-gnu/
sudo ln -s libboost_python-py34.so libboost_python3.so
```

and `selective_search_py` compiled.

**Comment by kevinmartinmauro** [comment]
thank you for updating

**Comment by kevinmartinmauro** [comment]
I thought I had solved this problem, but apparently I have not. I am wondering if you might have any idea where my libboost_python-py34.so might be installed? it is not in the x86_64-linux-gnu directory that you have it in. I don't even know which library this boost was installed from -- was it installed from Boost, or Boost.NumPy? It's hard for me to trace anything without more information.

 I do know that when I removed the "3" from lboost_python3, I can get the .so to compile. But then it doesn't know where anything is -- specifically, I get an error that says "error: scikit-image.filters --- no submodule named "filters""
